tests/common/libvm.sh

```diff
@@ -344,6 +344,24 @@ vm_get_journal_cursor() {
   vm_cmd journalctl -o json -n 1 | jq -r '.["__CURSOR"]'
 }
 
+# Wait for a message logged after $cursor matching a regexp to appear
+vm_wait_content_after_cursor() {
```

Comment:
Let's put a timeout on that and cleanly error out if it still didn't appear? That will at least allow the CI to collect the journal and upload it.